src/support/err.c

```diff
@@ -495,16 +519,21 @@ __wt_errx_func(WT_SESSION_IMPL *session, const char *func, int line, WT_VERBOSE_
   const char *fmt, ...) WT_GCC_FUNC_ATTRIBUTE((cold)) WT_GCC_FUNC_ATTRIBUTE((format(printf, 5, 6)))
   WT_GCC_FUNC_ATTRIBUTE((visibility("default")))
 {
+    char concatenated_id_str[WT_MAX_LOG_ID_LENGTH];
     va_list ap;
 
+    /* Use the default WT_DEFAULT_LOG_ID log ID '000' as default identifier for category-specific
```

Comment:
Multi line comment here.

Reply:
Fixed in 0f136343cc588b81524565f6e764cc7d7311d112